### ISSUE TITLE
Support BitBucket as addons source

### ISSUE THREAD

**Issue by achekulaev** [docksal/docksal#405]
Currently addons can only be installed from GitHub repositories. This is a ticket to add BitBucket support. Succeeds #368 

GitHub `https://raw.githubusercontent.com/$USER/$REPO/$BRANCH/$PATH/$FILE` 
BitBucket `https://bitbucket.org/$USER/$REPO/raw/$BRANCH/$PATH/$FILE`

**Comment by lmakarov** [comment]
See https://github.com/docksal/docksal/issues/368#issuecomment-348086947. I think should instead add support for arbitrary git repos (public or private).

**Comment by shelane** [comment]
Agreed. We have an on prem Bitbucket server where we would host ours.

**Comment by sean-e-dietrich** [comment]
@shelane we are currently looking at creating a way to include your own repositories in the PR https://github.com/docksal/docksal/pull/539 there have just been more pressing issues at the moment. But that is good to know this is a feature that others would like.
